test/general-tests/hardhat-run/check.sh

```diff
@@ -4,3 +4,4 @@ set -e
 
 npx hardhat starknet-compile contracts/contract.cairo
 npx hardhat run --no-compile scripts/deploy.ts
+npx hardhat run --no-compile scripts/quick-script.ts
```

Comment:
Now that we have this quick-script.ts, I think there's no need to be calling deploy.ts as well

Reply:
Yeah, makes sense.